DuosForSums.java

```diff
@@ -0,0 +1,49 @@
+import java.util.Arrays;
+
+public class DuosForSums {
+
+    public static void main(String[] args) {
+        int[] initArr = {1, 3, 5, -2, 6, 0, -3, -1, 7, 2};
+        int sumNum = 5;
+        group(initArr, sumNum);
+
+    }
+
+    private static void group(int[] arr, int x) {

```

Comment:
a little bit hard to understand but this is good enough for the current level. please take into account array could be created in this way new int[]{5,5}